### PR TITLE
Joe create and fill tables

### LINKED ISSUE #47: Create Employee Table
This module should create an ``Employee`` table 

### Process
- Write a function that drops the ``Employee`` table if it already exists
- Write a function that creates the ``Employee`` table if it doesn't already exist
- Insert the appropriate data into the table
- Export the function

### Expected Result
When the function is called in ``build-db.js``, it should create a table in the database with the data from the json files.

### LINKED ISSUE #48: Create Department Table
This module should create an ``Department `` table 

### Process
- Write a function that drops the ``Department `` table if it already exists
- Write a function that creates the ``Department `` table if it doesn't already exist
- Insert the appropriate data into the table
- Export the function

### Expected Result
When the function is called in ``build-db.js``, it should create a table in the database with the data from the json files.

### LINKED ISSUE #49: Create Computer Table
This module should create an ``Computer`` table 

### Process
- Write a function that drops the ``Computer `` table if it already exists
- Write a function that creates the ``Computer `` table if it doesn't already exist
- Insert the appropriate data into the table
- Export the function

### Expected Result
When the function is called in ``build-db.js``, it should create a table in the database with the data from the json files.

### LINKED ISSUE #52: Create Training_Program Table
This module should create an ``Training_Program`` table 

### Process
- Write a function that drops the ``Training_Program `` table if it already exists
- Write a function that creates the ``Training_Program`` table if it doesn't already exist
- Insert the appropriate data into the table
- Export the function

### Expected Result
When the function is called in ``build-db.js``, it should create a table in the database with the data from the json files.

### PR DESCRIPTION
**Pull request by joechesney**
# Description
I used Tim's amazing function to create 4 tables: Computers, Employees, Departments, Training Programs
I also destructured the funcitons in the build-db.js file

## Related Ticket(s)
Fixes #47 - Create Employees Table
Fixes #48 - Create Departments Table
Fixes #49 - Create Computers Table
Fixes #52 - Create Training Programs Table

## Problem to Solve
This creates 4 of the SQL database tables that we need

## Expected Behavior
if you run build-db.js it should create the 4 tables: Computers, Employees, Departments, Training Programs

## Steps to Test Solution

1. run 'node db/build-db.js'
2. check that the db was created INSIDE the db folder
3. open the database 

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

### CODE REVIEW

**Review by DavidLarsKetch — CHANGES_REQUESTED** [review]
Get those merge conflicts fiiiiiixed

**Review by kenziebottoms — DISMISSED** [review]

**Review by kenziebottoms — CHANGES_REQUESTED** [review]
Change `sqlRunTemplate.js:3` to:
```js
const db = new sqlite3.Database("db/api-sprint.sqlite");
```

File: db/products/makeCustomerTable.js [hunk 1]
```diff
@@ -1,6 +1,4 @@
 'use strict';
-const { generateSqlTable } = require("../sqlRunTemplate");
-const customers = require("../../data/json/customers.json");
```
Comment (DavidLarsKetch) [inline db/products/makeCustomerTable.js]: We need this. Shouldn't be deleted.

**Review by DavidLarsKetch — CHANGES_REQUESTED** [review]
Deleted `require("../sqlRunTemplate"` etc. from `makeCustomerTable.js`

**Review by kenziebottoms — APPROVED** [review]

**Review by jordan-castelloe — APPROVED** [review]
looks good!

**Pull request merged**